src/modules/displaymanager/main.py

```diff
@@ -782,6 +782,7 @@ class DMgreetd(DisplayManager):
     executable = "greetd"
     greeter_user = "greeter"
     greeter_group = "greetd"
+    greeter_css_location = "etc/greetd/style.css"
```

Comment:
Is this path and filename an official default for gtreetd?  If not, this should probably be set to `None` instead of being hard set to a path.

If it is an official path, it probably needs a leading `/` based on how you are using it.

Reply:
In that case, set it to `None`.  I don't think we should create an arbitrary default.

If they don't set it in the config file, we can not use it.